### ISSUE TITLE
System play/pause lifecycle events not called.

### ISSUE THREAD

**Issue by donmccurdy** [aframevr/aframe#3254]
**Description:**

Support for `play()`/`pause()` ([docs](https://aframe.io/docs/0.7.0/core/systems.html#methods)) were added recently (0.6.0?) but no longer seem to be working.

Tried:

1. Just registering a system.
2. Registering system, registering component, and using the component in the scene.

- A-Frame Version: 0.7.0
- Reproducible Code Snippet or URL: https://codepen.io/donmccurdy/pen/aVGJER?editors=1000

**Comment by mkungla** [comment]
@donmccurdy  I modified your example a little bit [codepen](https://codepen.io/mkungla/pen/OOZjjy).
I do agree that expected would be to see `system play` when `scene.play()` is called. 
 
However I never have seen actual usage `.play()` and `.pause()` in systems.  
Using `init()` or `update()` for setting up the system and adding custom methods which can be then called by different components like `this.system.doSomething()` has been more than enough for me and I have a feeling that `.play()` and `.pause()` in system context don not make much sense actually. 

Relevant commits:

docs [6b175ba586d9892b8f842a246506a0f92a44e0e7](https://github.com/aframevr/aframe/commit/6b175ba586d9892b8f842a246506a0f92a44e0e7)  
code [5a2ae56571e6bbc379d9acaba1912468ae690400](https://github.com/aframevr/aframe/commit/5a2ae56571e6bbc379d9acaba1912468ae690400)

**Comment by ngokevin** [comment]
I actually most of the time just use components versus systems. They can act like systems, have more features, and are more stable. Until we port systems to use component code and add features to systems that give them an edge over just using a component.

**Comment by donmccurdy** [comment]
> However I never have seen actual usage .play() and .pause() in systems.

Up until 0.6.0, none of the lifecycle events worked on systems — not even `update()`. Now, in 0.7.0, `play()`/`pause()` again do not work. So the fact that we haven't seen actual usage is not terribly surprising. Being able to use `pause()` in the physics system to stop physics simulation (running separately in a WW) would be an example use case.

> I actually most of the time just use components versus systems. They can act like systems, have more features, and are more stable. Until we port systems to use component code and add features to systems that give them an edge over just using a component.

We should fix this... we're doing the ECS model, systems should have functional lifecycle events. If `registerSystem` just becomes an alias for `registerComponent` that throws errors if you try to instantiate components anywhere but `<a-scene/>` that would be sufficient.

**Comment by ngokevin** [comment]
Yeah, we should fix. Just been on the backburner since it's too easily workaroundable. We can call it entity-component system vs entity-component-system :)
